README.md

```diff
@@ -34,37 +44,58 @@ Tags can be nested as demonstrated in the navigation snippet below.
 ```
 
 Keeping your HTML well indented so that every tag and "level" of nesting is
-aligned will make your code easier to read and maintain. Its aesthetics, how it
-looks, is conveying its structure.
+aligned will make your code easier to read and maintain.
 
 ### HTML Attributes
 
-HTML attributes represent metadata about the containing element.  The
-attributes can be used to identity, classify, style or modify the default
-behavior of the element.
+An HTML attribute is extra information we can add to a tag to identify,
+classify, style or modify the default behavior of the element the tag contains.
 
 ```html
-	<element attribute_name=attribute_value another_attribute_name=another_attribute_value></element>
+  <element attribute_name=attribute_value another_attribute_name=another_attribute_value></element>
 ```
+A common use case for HTML attributes is element identification (`id`
+attribute) and/or classification (`class` attribute). The `id` attribute is used
+to uniquely identify an element within the whole document. The `class` attribute
+is used to group together similar elements.
 
-A very common use case for HTML attributes is element identification (`id`
-attribute) and/or classification (`class` attribute). The `id` attribute is
-used to uniquely identify an element within the whole document. The `class`
-attribute is used to group together similar elements.
+Both `<id>` and `<class>` attributes are often used for styling purposes since
+they allow us to find a specific element or style similar elements with a
+single style declaration.
+
+```html
+  <p id='main_paragraph'>This element can be uniquely identified  using the 'main_paragraph' id HTML attribute</p>
+  
+  <p class='other_paragraphs'>This element belongs to a group of elements who share the 'other_paragraphs' HTML class attribute</p>
+  <p class='other_paragraphs'>This element also belongs to a group of elements who share the 'other_paragraphs' HTML class attribute</p>
+```
 
-Both `<id>` and `<class>` attributes are often used for styling purposes since they allows us to find a specific element or style similar elements with a single style declaration.
+Attributes that identify or classify are not necessary for the HTML
```

Comment:
I like calling out the difference between necessary attributes and optional attributes. Good addition to the lesson to help students understand all attributes are not created equal.